Jenkinsfile

```diff
@@ -97,6 +97,26 @@ void pushLogFile(String FILE_NAME) {
     }
 }
 
+void pushK8SLogs(String TEST_NAME) {
+    def LOG_FILE_PATH="e2e-tests/logs"
+    echo "Push k8s logs to S3!"
+
+    withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', accessKeyVariable: 'AWS_ACCESS_KEY_ID', credentialsId: 'AMI/OVF', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {
+        sh """
+            if [ -d "${LOG_FILE_PATH}/${TEST_NAME}" ]; then
+                env GZIP=-9 tar -zcvf ${TEST_NAME}.tar.gz -C ${LOG_FILE_PATH} ${TEST_NAME}
```

Comment:
Should not here be a slash? `${LOG_FILE_PATH}/${TEST_NAME}`

Reply:
no, because it's two separate parameters, `-C ${LOG_FILE_PATH}` changes active directory and  `${TEST_NAME}` is another parameter